lib/surface/compiler.ex

```diff
@@ -317,6 +318,39 @@ defmodule Surface.Compiler do
      }}
   end
 
+  defp convert_node_to_ast(
+         :unless,
+         {_, attributes, children, node_meta},
+         compile_meta
+       ) do
+    meta = Helpers.to_meta(node_meta, compile_meta)
+
+    default = %AST.AttributeExpr{value: false, original: "", meta: node_meta}
+
+    condition = attribute_value_as_ast(attributes, "condition", default, compile_meta)
+
+    inverted_condition =
+      case condition do
+        %AST.AttributeExpr{value: value} = expr ->
+          value =
+            quote generated: true do
+              !unquote(value)
+            end
```

Comment:
@msaraiva @lnr0626 so this works, but I need a sanity check if this is the right way to go :)